libs/ui/src/invalid_card_screen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Screen } from './screen';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+export function InvalidCardScreen(): JSX.Element {
+  return (
+    <Screen white>
+      <Main centerChild>
+        <Prose textCenter theme={fontSizeTheme.medium} maxWidth={false}>
```

Comment:
Any reason we apply `maxWidth={false}` to this screen but not the `RemoveCardScreen`?

Reply:
It looks like the `InvalidCardScreen` text will wrap if `maxWidth` is true. But the `RemoveCardScreen` text is too short to wrap even if `maxWidth` is true.

Reply:
Got it!